encoders/nlp/TextPaddlehubEncoder/tests/test_textpaddlehubencoder.py

```diff
@@ -0,0 +1,67 @@
+import numpy as np
+import os
+import mock
+import shutil
+
+from .. import TextPaddlehubEncoder
+from jina.executors import BaseExecutor
+
+
+target_output_dim = 1024
+test_data = np.array(['it is a good day!', 'the dog sits on the floor.'])
+tmp_files = []
+
+
+def teardown():
+    print('tear down...')
+    for k in tmp_files:
+        if os.path.exists(k):
+            if os.path.isfile(k):
+                os.remove(k)
+            elif os.path.isdir(k):
+                shutil.rmtree(k, ignore_errors=False, onerror=None)
+
+
+def add_tmpfile(*path):
+    tmp_files.extend(path)
+
+
+class MockModule:
+    def __init__(self):
+        print('i am a mocker module')
+
+    def get_embedding(self, texts, *args, **kwargs):
+        print('i am a mocker embedding')
+        return [[np.random.random(target_output_dim), None]] * len(texts)
+
+
+@mock.patch('paddlehub.Module', return_value=MockModule())
+def test_textpaddlehubencoder_encode(mocker):
```

Comment:
is there a chance to add an extra test with a real model?

Reply:
Yes, I plan to set two types of test. One is for the cicd process, which use mocks for testing. The other is for building the Docker image, in which the real model will be loaded.